### PR TITLE
fixes the encode test

### PR DESCRIPTION
**Pull request by thathert**

### CODE REVIEW

**Review by ddietterich — APPROVED** [review]
LGTM
I agree with your bias to being very explicit. It will probably cost us failures again as we track BQ changes.

**Comment by ddietterich** [comment]
Is that scope creep or fixes?

**Comment by thathert** [comment]
(unfortunately) All necessary to fix integration. I think it is time to revisit how we drop tables and apply migrations. We keep ending up in a locked state in the database and I need to go resolve manually. 

Scenario: we run migrations every time an application context spins up. This happens many times during tests. If a container gets shut down during any test, such as when a new change is pushed to a PR, there is a not-so-small chance that the container had a lock that it will never release.

The next container that comes in politely keeps waiting for the other (now gone) container to release the lock. I've been having to go in and manually release the lock -- hence the db-connect.sh script. Useful for more than just this, but obviously not the long term solution.

I have a piece of code from the rename branch that I want to split out into a separate PR to address this. It adds a "lockTimeoutMins" property that will inform a container how long it should wait on a lock before it will force release the lock to unblock itself.

Separately, I'm beginning to think that migrations should be run at deploy-time, not at app-context-creation time. It will help speed up tests, and should reduce the number of times we end up with zombie locks :)

File: src/test/java/bio/terra/filesystem/EncodeFileTest.java
Comment (RoriCremer) [inline src/test/java/bio/terra/filesystem/EncodeFileTest.java]: wait, is this a pun I don't get or a typo?

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**
